README.md

```diff
@@ -30,6 +34,7 @@ jobs:
     permissions:
       id-token: write
       contents: read
+      pull-requests: write
```

Comment:
`@guardian/actions-static-site` v3 needs write permission for the pull-requests as it passes it to `@guardian/actions-riff-raff` v4 which requires this permission.